examples/visualisation/visualise_data.ipynb

```diff
@@ -19,13 +21,64 @@
     "### `visualization`\n",
     "contains utilities to draw additional information (e.g. trajectories) onto RGB images. These utilities are commonly used after a `to_rgb` call to add other information to the final visualisation. \n",
     "One example is:\n",
-    "- `draw_trajectory`: this function draws 2D trajectories from coordinates and yaws offset on an image\n"
+    "- `draw_trajectory`: this function draws 2D trajectories from coordinates and yaws offset on an image\n",
+    "\n",
+    "*By using L5 datasets, you accept the license at https://self-driving.lyft.com/level5/download/ and the licenses present inside of each dataset archive.*"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "cellView": "form",
+    "id": "-2bOYHKU1tw2"
+   },
+   "outputs": [],
+   "source": [
+    "#@title Download L5 Sample Dataset and install L5Kit\n",
+    "!rm -rf l5kit_dataset\n",
+    "\n",
+    "# Download sample zarr\n",
+    "print(\"Downloading sample zarr dataset...\")\n",
+    "!wget https://lyft-l5-datasets-public.s3-us-west-2.amazonaws.com/prediction/v1.1/sample.tar -q --show-progress\n",
+    "!tar xf sample.tar\n",
```

Comment:
any chance we can get a tmp dir here?

Reply:
Let's have a sync about that because since there are other notebooks I was thinking of moving this functionality to a Python code inside of the l5kit module.